portfolio/src/main/webapp/index.html

```diff
@@ -138,9 +155,9 @@ <h4>Robotic Systems Team Lead | Sep 2018 - May 2019</h4>
         <section id="contact" class="content">
           <h2>Get In Touch</h4>
           <p>...</p>
-          <a href="mailto:zachary.ghera@gmail.com" id="link-box">Say Hi</a>
+          <a style="padding-bottom:20vh" href="mailto:zachary.ghera@gmail.com" id="link-box">Say Hi</a>
```

Comment:
Is this "link-box" ID used anywhere? It's probably better to keep all styling together in the CSS file, rather than mixing between HTML and CSS files. You could use the link-box ID to apply the padding-bottom style in the CSS file, and remove the "style" attribute here. If you think it's better with the styling here, then remove the unused ID instead.

Reply:
See commit 6a6fd58.  tldr: removed this link entirely.

> Remove link to email in Contact section.
> 
> This change is in response to comment about an unused CSS id. This link was taken out in a latter branch commit as the sidebar already provides access to my email to contact me. A latter addition will include an arrow to the sidebar or pull the sidebar icons to the bottom regardless of the window side when the user scrolls to the 'Contact' page.